app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1511,6 +1544,7 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
     }
 
     private fun waitForConnection() {
+        // Called inside an this.loadUrl() [not webView.loadUrl()]
```

Comment:
This is called from other functions as well? I don't really understand the comment and there seems to be broken Markdown syntax as well.

Reply:
This is not Markdown. This is a pointer to line 1263.

Reply:
This is a note:
```kotlin
// BAD:
this.loadUrl()
this.waitForConnection()

// Good:
this.loadUrl()

// Good:
webView.loadUrl()
this.waitForConnection()
```